components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -330,28 +331,29 @@ public void removeExpiredSessionData(Timestamp timestamp) {
             log.error(e.getMessage(), e);
             return;
         }
+        long cleanupLimit = TimeUnit.MINUTES.toNanos(FrameworkUtils.getCurrentStandardNano() - IdentityUtil
```

Comment:
Should be corrected as,

`long cleanupLimitNano = FrameworkUtils.getCurrentStandardNano() - TimeUnit.MINUTES.toNanos(IdentityUtil.getCleanUpTimeout()));`